res/res_config_curl.c

```
@@ -607,15 +607,12 @@ static int reload_module(void)
 	struct ast_config *cfg;
 	struct ast_variable *var;
 
-	if (!(cfg = ast_config_load("res_curl.conf", flags))) {
-		return 0;
-	} else if (cfg == CONFIG_STATUS_FILEINVALID) {
-		ast_log(LOG_WARNING, "res_curl.conf could not be parsed!\n");
+	cfg = ast_config_load("res_curl.conf", flags);
+	if (!cfg || cfg == CONFIG_STATUS_FILEINVALID) {
```

Usually we warn about FILEINVALID so might make sense to keep warning in the case the config is malformed, rather than silently ignore?

---

`config_text_file_load` already prints an error when it can't parse the file:

```
[Mar 17 12:32:52] WARNING[89739]: config.c:2279 process_text_line: parse error: No category context for line 4 of /etc/asterisk/res_curl.conf
```

Is that not sufficient?

---

> `config_text_file_load` already prints an error when it can't parse the file:
> 
> ```
> [Mar 17 12:32:52] WARNING[89739]: config.c:2279 process_text_line: parse error: No category context for line 4 of /etc/asterisk/res_curl.conf
> ```
> 
> Is that not sufficient?

Ah, I didn't realize that. Then yeah, looks good to me!